README.md

```diff
@@ -1 +1,103 @@
 # terraform-aws-ecs-scheduled-task
+
+A module to supports the ability to schedule tasks on either a cron -like schedule or in a response to CloudWatch Events in ECS.
```

Comment:
We are not actually do: 
"in a response to CloudWatch Events in ECS".

Probably delete the words?

Reply:
done.